### PR TITLE
Login

### PR DESCRIPTION
**Pull request by maxxof**
- User can login using correct credentials
- System saves user session in local storage
- Homepage

### CODE REVIEW

File: src/components/HomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+const HomePage = (props) => {
+  const navigate = useNavigate();
+  useEffect(() => {
+    if (!props.user) {
+      navigate('/login')
+    }
+  }, [])
+
+  return !props.user
+    ? navigate('/login')
```
Comment (PyryL) [inline src/components/HomePage.jsx:15]: maybe not return the result of `navigate` which [seems to be](https://reactrouter.com/en/main/hooks/use-navigate#usenavigate) `void`?

File: src/components/Register.jsx
Comment (nuclearkittens) [inline src/components/Register.jsx:1]: fix imports:
+ rows 2, 8 >> ```import { useState, useEffect } from 'react';```
+ row 3, 4 >> ```import { Link, useNavigate } from '../Router';```

**Review by PyryL — CHANGES_REQUESTED** [review]

File: src/controllers/sessionController.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const createSession = (userData) => {
+    window.localStorage.setItem(
+      'loggedRuokasovellusUser', JSON.stringify(userData)
```
Comment (nuclearkittens) [inline src/controllers/sessionController.js:3]: ```loggedRuokasovellusUser``` is not the most descriptive name

**Review by nuclearkittens — CHANGES_REQUESTED** [review]
couple of fixes needed, otherwise ok

**Review by nuclearkittens — APPROVED** [review]
ok!

**Review by PyryL — APPROVED** [review]

**Review by PyryL — APPROVED** [review]
looks good

**Pull request merged**